docs/src/rules/no-invalid-this.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: no-invalid-this
 rule_type: suggestion
+handled_by_typescript: true
+extra_typescript_info: Note that technically, TypeScript will only catch this if you have the "strict" or "noImplicitThis" flags enabled. These are enabled in most TypeScript projects, since they are considered to be best practice.
```

Comment:
```suggestion
extra_typescript_info: Note that, technically, TypeScript will only catch this if you have the `strict` or `noImplicitThis` flags enabled. These are enabled in most TypeScript projects, since they are considered to be best practice.
```

Reply:
I deliberately used quotes instead of backticks here because they don't propagate properly through to the markdown template - it results in the literal backtick instead of code. I also tried using `<code>`, which results in the same thing, a literal `<code>`. Thus, I recommend using quotes for now, unless you know a specific way around that.